src/rebar_raw_resource.erl

```diff
@@ -426,9 +435,10 @@ make_vsn(Path) ->
 %
 make_vsn(AppInfo0, _ResourceState) ->
     Spec = case rebar_app_info:source(AppInfo0) of
-        {?RTYPE, Loc0, #mod_ref{res = Res0, ref = Ref}} -> {Res0, Loc0, Ref};
-        {?RTYPE, S, _} -> S;
-        {?RTYPE, S} -> S
+               {?RTYPE, Loc0, #mod_ref{res = Res0, ref = Ref}, SubLoc} -> {Res0, Loc0, Ref, SubLoc};
+               {?RTYPE, Loc0, #mod_ref{res = Res0, ref = Ref}} -> {Res0, Loc0, Ref};
+               {?RTYPE, S, _} -> S;
+               {?RTYPE, S} -> S
     end,
```

Comment:
As long as you're re-indenting to align with the `case` here, shouldn't the `end` be indented to line up with the `case` as well?

Reply:
emacs aligned the code this way...